### DISCUSSION TITLE
Why is there a new release for every change of the README file?

### DISCUSSION THREAD

**Discussion by kirenida** [dtcooper/raspotify#622] [General]
What it says in the title - why is there a new release every time something changes in the readme file?

Shouln't a new release imply some sort of meaningful change in the software, like a bugfix or new functionality, instead of just a few deleted / added characters / lines in the readme?

This is not an actual problem in any way, I'm just curious as to why this is done, as I assume most users just read the instructions online.

**Comment by JasonLG1979** [comment]
See https://github.com/dtcooper/raspotify/issues/600

I've been meaning to do a a changelog so it seems more obvious what's going on.

**Reply by kirenida** [reply]
ok, so what I understand from #600 , even if it says the only change in Raspotify is in the README file, there may be changes in librespot that gets installed at the same time?

if so, then yes, it would be a good idea to mention those changes, or at least state that a new release is because of changes in librespot.

**Reply by JasonLG1979** [reply]
Yep. This repo is only build scripts and files need to turn a librespot binary into a systems service. The librespot source does not live here.
